src/main/java/io/appform/jsonrules/expressions/preoperation/date/CalendarOperation.java

```diff
@@ -60,5 +67,6 @@ public Number compute(ExpressionEvaluationContext context) {
 	}
 
 	protected abstract Number compute(JsonNode evaluatedNode, String operand, String zoneOffSet);
```

Comment:
Two similar abstract functions are not necessary. Remove the other implementation, make this final and delegate the call to new function with the default pattern